app/test_split_integer.py

```diff
@@ -2,20 +2,21 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    summa = sum(split_integer(17, 4))
+    assert summa == 17
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(17, 4) == [4, 4, 4, 5]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(8, 1) == [8]
```

Comment:
The test function `test_should_return_part_equals_to_value_when_split_into_one_part` is correct, but it would be better to add more tests for this case with different numbers. This is a recommendation to improve test coverage.